demos/howtos/myula.py

```diff
@@ -0,0 +1,289 @@
+"""
```

Comment:
This tutorial is super nice. It has some code style things like the "=" not having spaces from variable definition to code. I suggest running it through a code formatter like "black". https://black.vercel.app/

Reply:
Thanks for your comment. We formatted the code as you suggested.